notebooks/classification_2d_mednist.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<p>does wget also work on windows? Otherwise our tests will fail..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>you are right</p>